### PR TITLE
Kirill.sitcevoy/unit test

### PR DESCRIPTION
**Pull request by kievzenit**
Unit tests hometask

### CODE REVIEW

File: UnitTest/UnitTestTests/StringExtensionTests.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using UnitTest;
+
+namespace UnitTestTests
+{
+    [TestClass]
+    public class StringExtensionTests
+    {
+        [TestMethod]
+        public void Handle_When_ColorIsBlack_Then_MustBeTrue()
```
Comment (Artya) [inline UnitTest/UnitTestTests/StringExtensionTests.cs]: Why Handle, if your action is IsBaseColor?

Comment (Artya) [reply UnitTest/UnitTestTests/StringExtensionTests.cs]: Better to name it as `IsBaseColor_When_ColorIsBlack_Then_ShouldReturnTrue`

File: UnitTest/UnitTestTests/StringExtensionTests.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        {
+            // Arrange
+            const string color = "black";
+
+            // Act
+            bool actual = color.IsBaseColor();
+
+            // Assert
+            const bool expected = true;
```
Comment (Artya) [inline UnitTest/UnitTestTests/StringExtensionTests.cs]: this one should be in arrange section, all setup of the test and expected results should be in arrange

File: UnitTest/UnitTestTests/StringExtensionTests.cs (hunk 1 above)
Comment (kievzenit) [reply UnitTest/UnitTestTests/StringExtensionTests.cs]: Done in 3b0d4f72e4804b22ea2b3b5437831f9e099c7dd2

File: UnitTest/UnitTestTests/StringExtensionTests.cs (hunk 2 above)
Comment (kievzenit) [reply UnitTest/UnitTestTests/StringExtensionTests.cs]: Done in 3b0d4f72e4804b22ea2b3b5437831f9e099c7dd2

**Pull request closed without merging**